cmd/crypto/metadata.go

```diff
@@ -77,6 +77,12 @@ func (ssec) IsEncrypted(metadata map[string]string) bool {
 	if _, ok := metadata[SSECSealedKey]; ok {
 		return true
 	}
+	if _, ok := metadata[SSEIV]; ok {
```

Comment:
Again, not correct. That function would return true for a SSE-S3 encrypted object to. That was done on purpose.

Reply:
agreed - removed these as well.